tanda/tan.py

```diff
@@ -233,14 +258,18 @@ def get_transformed_data(self, session, data, emit_incremental=False,
             data_rep, tf_seqs, emit_incremental=emit_incremental
         ), tf_seqs, data_rep)
 
-    def transform(self, session, x):
+    def transform(self, session, x, use_cache=True):
         """Transform single data point
-            @session: a TensorFlow session
-            @x:       original training data point
+            @session:   a TensorFlow session
+            @x:         original training data point
+            @use_cache: use TF seq cache?
         Returns a transformed data point
         """
         # Get action sequences
-        tf_seq = self.generator.get_action_sequence(session, 1)[0, :]
+        if use_cache:
```

Comment:
Why not just check if the cache exists rather than requiring a second kwarg?

Reply:
Also this defaults to `True` which will break if no cache... I'll make this change